From 13c1d3e4b4aebe7f70109a290dad95bb2cbce247 Mon Sep 17 00:00:00 2001
From: Alexander Shiyan <shc_work@mail.ru>
Date: Sat, 26 Apr 2014 08:52:06 +0400
Subject: [PATCH 079/110] ARM: dts: imx27-phytec-phycore-rdk: Remove duplicate
 entries for USB PHY

"compatible", "#address-cells" and "#size-cells" for USB PHY are
already described in the SOM DTS. Remove these duplicate entries.

Signed-off-by: Alexander Shiyan <shc_work@mail.ru>
Signed-off-by: Shawn Guo <shawn.guo@freescale.com>
---
 arch/arm/boot/dts/imx27-phytec-phycore-rdk.dts | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx27-phytec-phycore-rdk.dts b/arch/arm/boot/dts/imx27-phytec-phycore-rdk.dts
index 0875327..ac18ccf 100644
--- a/arch/arm/boot/dts/imx27-phytec-phycore-rdk.dts
+++ b/arch/arm/boot/dts/imx27-phytec-phycore-rdk.dts
@@ -39,10 +39,6 @@
 	};
 
 	usbphy {
-		compatible = "simple-bus";
-		#address-cells = <1>;
-		#size-cells = <0>;
-
 		usbphy2: usbphy@2 {
 			compatible = "usb-nop-xceiv";
 			reg = <0>;
-- 
1.9.2

